SAST-JP/c101007058.lua

```diff
@@ -0,0 +1,79 @@
+--Trickstar Live Stage
+--Logical Nonsense
+
+--Substitute ID
+local s,id=GetID()
+
+function c2819435.initial_effect(c)
```

Comment:
Using an ID here instead of s

Reply:
Uhhh...about that...I deleted my fork. Oops.

Reply:
It's fine. I'd merge and fix it myself, but somehow there's a conflict, so instead I'll fix it locally and upload that copy